src/data/gallery.js

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable camelcase */
+import india2014_1 from "public/trips/india2014/2014_1.webp";
```

Comment:
<img width="675" alt="image" src="https://github.com/acm-ucr/rishi-website/assets/90938120/6e0b6da7-fbf9-42f3-a220-1ec7b5a9b2f2">

include a @/ before each import

u can see if your imports work doing something like this

<img width="573" alt="image" src="https://github.com/acm-ucr/rishi-website/assets/90938120/5ca983cc-eb16-4da2-8d50-ddd7835e8332">